InnoSetup/CnCNet5_YR_Installer.iss

```diff
@@ -1,6 +1,6 @@
 ;Made with Inno Setup 5.5.3 Ansi
 #include <.\Inno Download Plugin\idp.iss>
-#include <.\ISTheme\ISTheme.iss>
+;#include <.\ISTheme\ISTheme.iss>
```

Comment:
Remove the theme for now, until we want to spend time in updating/figuring out the error messages thing.